client/src/document/index.tsx

```diff
@@ -143,6 +144,11 @@ export function Document(props /* TODO: define a TS interface for this */) {
 
   const isServer = typeof window === "undefined";
 
+  let retiredLocale = false;
+  if (!isServer) {
+    retiredLocale = document.location.search.indexOf("retiredLocale") > -1;
+  }
```

Comment:
There's a hook for that. 
```javascript
const { retiredLocale } = useParams();
```
...might work.

Reply:
Ah, interesting. Will take a look.

Reply:
I tried using this but:

1. You cannot use hooks in a condition (you know this of course :))
2. Attempted to change `{locale} = useParams();` to `{locale, retiredLocale} = useParams();` but it is always undefined. Perhaps `useParams` only works with router level params?